### DISCUSSION TITLE
Empty list when retrieving verifiable credentials.

### DISCUSSION THREAD

**Discussion by danimaraas** [walt-id/.github#73] [Help] [answered]
I am trying out the SSI kit, and have a question concerning the first tutorial "My First VC". I played around with it and followed the steps. Each step seemed to work fine, but afterwards I was surprised by the behaviour of some of the endpoints. The dids were retrievable, but not the Verifiable Credential I created. The endpoint returned an empty list. Curious,  I attempted to verify the VC a second time (I copied the initial response when creating the VC), and now it returned false. Assuming that the mistake was on my end, I repeated the entire tutorial (except for creating the dids). Again, everything seems to work fine. I get a response with the credential, and I'm, able to verify it. However, when I try to retrieve the VC afterwards, I get an empty list. Any good suggestions as to why I get an empty list?

**Comment by taminobaumann** [comment, marked as answer]
Hey @danimaraas, 

thanks for reaching out. 

By default, the SSI-Kit does not store the credential it signs/issues. However, you can use the [Auditor API](https://docs.walt.id/v/ssikit/getting-started/rest-apis/custodian-api/credentials-management#store-credential) to store the credential after it has been signed for later retrieval. 

Hope this helps! 

Best
Tamino

**Reply by danimaraas** [reply]
Thank you for clearing that out for me!
